src/app/side-nav/side-nav.component.ts

```diff
@@ -51,16 +50,15 @@ export class SideNavComponent implements OnInit, AfterViewChecked {
     if (window.scrollY >= sectionTop - 215 && window.scrollY != height) current = `${section.getAttribute("id")}`;
     });
     //set current to lowest section is scroll is at bottom of content
-    if (window.scrollY === height) current = `${sideNavTitles[sideNavTitles.length - 1].getAttribute("id")}`;
+    if (window.scrollY >= height) current = `${sideNavTitles[sideNavTitles.length - 1].getAttribute("id")}`;
```

Comment:
>==

Reply:
Well, it's autoformatting, but the idea is you need a true statement, not a truethy one.